stable/database/files/nuolatency.py

```diff
@@ -0,0 +1,54 @@
+import socket, pprint
+from time import time
+
+from pynuoadmin import nuodb_cli
+
+def latency_point(host, port = 48004, timeout = 5):
+    """
+    Calculate a latency point using sockets. If error return large number
+    """
+
+    # New Socket and Time out
+    s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    s.settimeout(timeout)
+
+    # Start a timer
+    s_start = time()
+
+    # Try to Connect
+    try:
+        s.connect((host, int(port)))
+        s.shutdown(socket.SHUT_RD)
+    except:
+        return 99999
```

Comment:
Do you want the returned latency to be large that we do not use the admin that generated the exception? I would just propagate the exception and let the loop that iterates over the admins catch the exception and skip it.

If you don't want to remove the admin from consideration completely in case the others also generate exceptions, then I guess you could do this, but it would be tricky to distinguish between admins that have high latency (higher than 5 seconds) and ones that are completely unresponsive.

Reply:
Yes it probably does not help to return a large number and keep iterating.

Reply:
@adriansuarez  I pushed a change that throws exception and then handle the exception and exit the loop with 99999